codalab/apps/web/static/js/search/ws_actions.js

```diff
@@ -98,6 +98,15 @@ var WorksheetActions =  function() {
                                 if (data.data.stdout){
                                     console.log(data.data.stdout);
                                     term.echo(data.data.stdout);
+                                    var references = data.data.structured_result['refs'];
+                                    Object.keys(references).forEach(function(k) {
+                                        $(".terminal-output div div:contains("+ k + ")").html(
```

Comment:
For consistent style I would use what you did with the anonymous function to the `forEach` call:
```
$(".terminal-output div div:contains(" + k + ")").html(function(idx, html) {
    ...
});
```
which happens to be pretty standard.